docker/start.sh

```diff
@@ -7,11 +7,10 @@ cat <<EOF > /opt/docker/conf/local-dev.conf
 play.http.secret.key="$key"
 
 app.webgroup.prefix="in-"\${app.name.id}"-local-dev-"
-play.filters.hosts {
-  allowed = ["localhost:8090", "localhost:8443"]
-}
+
+play.filters.disabled+=play.filters.hosts.AllowedHostsFilter
```

Comment:
```suggestion
play.filters.disabled += play.filters.hosts.AllowedHostsFilter
```

Reply:
Fixed in https://github.com/UniversityofWarwick/sso-stub/pull/13/commits/829b46d2979e6d92091fc19eda6e0f131aa1bf9e